This is a patch to get sources for the maintenance update 5976 (Rtools43)
from svn revision 5975.

The update upgrades postgresql to 13.20 to fix CVE-2025-1094 in PostgreSQL.

Index: toolchain_libs/mxe/src/postgresql.mk
===================================================================
--- toolchain_libs/mxe/src/postgresql.mk	(revision 5975)
+++ toolchain_libs/mxe/src/postgresql.mk	(working copy)
@@ -4,8 +4,8 @@
 $(PKG)_WEBSITE  := https://www.postgresql.org/
 $(PKG)_DESCR    := PostgreSQL
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 13.13
-$(PKG)_CHECKSUM := 8af69c2599047a2ad246567d68ec4131aef116954d8c3e469e9789080b37a474
+$(PKG)_VERSION  := 13.20
+$(PKG)_CHECKSUM := 8134b685724d15e60d93bea206fbe0f14c8295e84f1cc91d5a3928163e4fb288
 $(PKG)_SUBDIR   := postgresql-$($(PKG)_VERSION)
 $(PKG)_FILE     := postgresql-$($(PKG)_VERSION).tar.bz2
 $(PKG)_URL      := https://ftp.postgresql.org/pub/source/v$($(PKG)_VERSION)/$($(PKG)_FILE)
